scalafix-rules/src/main/scala/scalafix/internal/rule/RemoveUnused.scala

```diff
@@ -143,6 +143,8 @@ class RemoveUnused(config: RemoveUnusedConfig)
           }.asPatch
         case i: Defn if isUnusedTerm(i.pos) =>
           defnTokensToRemove(i).map(Patch.removeTokens).asPatch.atomic
```

Comment:
Keeping the old matcher for compatibility with semanticdb generated with compiler <2.13.12.